src/sidebar/components/test/annotation-test.js

```diff
@@ -116,10 +131,11 @@ describe('annotation', function() {
     before(function() {
       angular.module('h', [])
         .component('annotation', annotationComponent())
+        .component('annotationActionButton', {
+          bindings: require('../annotation-action-button').bindings,
+        })
         .component('markdown', {
-          bindings: {
-            customTextClass: '<?',
-          },
+          bindings: require('../markdown').bindings,
```

Comment:
Yeah that's a good property. It's a shame that when `require`d dependencies are mocked using `proxyquire`, `proxyquire` doesn't seem to have any capability to do the same thing. (Python's `mock` can automatically scan a class and make a mock version of it that has the same methods and arguments.)